### ISSUE TITLE
[New build - EXPERIMENTAL] 21w10b: Pipes

### ISSUE THREAD

**Issue by tsunamayo** [tsunamayo/Starship-EVO#3659]
This build bring a few improvements to fuel pipes:
 - The see through preview has been toned down, and will only display close to the camera and near the center of the screen.
 - It is possible to deactivate the see through inside the F3 menu.
 - A pipe 6-way junction brick as been added in small and large.
 - The codex tabs have been shuffled a bit, now inside the pipe tab will be only bricks compatible with the upcoming fuel logic.
 
 Bugfixes:
 - Performance issues on the latest multi-threaded code.

**Comment by tsunamayo** [comment]
Build is live!

**Comment by tsunamayo** [comment]
I have just noticed shield are broken - I will push a new patch asap.

**Comment by tsunamayo** [comment]
Fixed.

**Comment by Kaiser-Indrasil** [comment]
I love that you've put the most importand debug options at the top <3
![image](https://user-images.githubusercontent.com/63756057/110801780-79bac580-8275-11eb-9d52-85f1349eb31c.png)

Both 6-way connectors are invisible and have a missing description/name on the HUD. I love the shape though!
![image](https://user-images.githubusercontent.com/63756057/110802078-c0a8bb00-8275-11eb-8f58-d6a62f4b6e42.png)

I've noticed you changed the colour of the connection points on the Handle and the Ladder and imho, they looked better where they were black/very dark grey. There is simply no dark shade like that in the colour palette to match these connection points.
![image](https://user-images.githubusercontent.com/63756057/110802421-1715f980-8276-11eb-92fb-4e9781d18f9b.png)

Also, all 3 of the Round Tiles, including the thin one, count as pipes both in the Codex, and in the Overlay. Does that mean, they function as pipes? Cos that would be awesome.
![image](https://user-images.githubusercontent.com/63756057/110803009-a91e0200-8276-11eb-9d6f-c6854ad48e78.png)

**Comment by TIKIRobo** [comment]
Yea those new connectors are invisible but their shape from what i can tell looks pretty good

**Comment by Briaireous** [comment]
Hell yeah, thats some banana bread right there.

**Comment by RockefellerDoctor** [comment]
a LOT of ships are gonna have to be rebuilt once this hits the Default build.

**Comment by Briaireous** [comment]
Indeed, I may have to add some piping in my city modules to hopefully transport fuel through out the cities.

**Comment by Uncle-Ulty** [comment]
> Indeed, I may have to add some piping in my city modules to hopefully transport fuel through out the cities. Depending on how fuel use will be like

this is one of the reasons I made the subway track so deep in the ground, to save space for future features
